test/various.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var request = require('../');
+var t = require('chai').assert;
+
+describe('Various', function () {
+    it('should work with Basic Authentication', function (done) {
```

Comment:
could we split this file in two, one for "test/auth.test.js" another one for "test/cookie.test.js" ?